Server/include/BaseQuasarServer.h

```diff
@@ -99,10 +99,17 @@ class BaseQuasarServer
     int parseCommandLine(int argc, char *argv[], bool *isHelpOrVersion, bool *isCreateCertificateOnly, std::string *configurationFileName, std::string& opcUaBackendConfigurationFile);
     //initialize XML parser, logging and UA stack (in that order)
     int initializeEnvironment();
+    //Dump calling environment, cmd line args etc. to Log file
+    void logEnvironment() const;
+    // Gets the current process environment variables
+    std::string getProcessEnvironmentVariables() const;
+    // Gets the current process owner
+    std::string getProcessOwner() const;
     //Cleans the server before exiting.
     void shutdownEnvironment();
     //Handler for initializing the node manager configuration only when the server is ready
     UaStatus configurationInitializerHandler(const std::string& configFileName, AddressSpace::ASNodeManager *nm);
 
+    std::list<std::string> m_commandLineArgs;
```

Comment:
What is the reason for storing this information? Can't it be simply printed and forgotten?

Reply:
I did waver about this. Facts is we cannot log command line args until after LogIt is initialised. So args need to be kept around somehow.
_Could_ pass args down through method parameter lists until logEnvironment is called (after LogIt initialised), but that felt cheesy too: requires extending parameter list of a few methods.
Happy to go via the parameter list route if you think that's a better way to go.

Reply:
Hi @pnikiel,
So - all other comments resolved: just this one remains. As I said, happy to change the impl to pass the info through method parameters if preferred. I felt storage option gives cleaner code, albeit at small memory price - bit no big deal either way.

BTW: you'll need to upmerge to nebula for working build - boost_filesystem gone in latest feature branch.